src/main/java/com/vinylteam/vinyl/service/impl/DefaultPasswordRecoveryService.java

```diff
@@ -23,88 +23,106 @@
 @Slf4j
 @Service
 @RequiredArgsConstructor
-@PropertySource("classpath:application.properties")
 public class DefaultPasswordRecoveryService implements PasswordRecoveryService {
     private static final String RECOVERY_MESSAGE = "Hello, to change your password, follow this link: \n{applicationLink}/recoveryPassword/newPassword?token=";
-
-    private final PasswordRecoveryDao passwordRecoveryDao;
-    private final UserService userService;
-    private final MailSender mailSender;
     @Value("${application.link}")
```

Comment:
You could use static constants for such declarations
Actually the point of @Value is to have its actual value specified in a property file
You could also use : operator inside @Value so that default value will be used if specified key does not exist in a property file

`@Value("${application.link:default-url}")`